CONTRIBUTING.md

```diff
@@ -1,15 +1,25 @@
 [Back to README](README.md)
```

Comment:
Can we please remove this. Doesn't look nice. Thanks

Reply:
ok